src/lightning_app/core/flow.py

```diff
@@ -93,6 +93,7 @@ def __init__(self):
         """
         from lightning_app.runners.backends.backend import Backend
 
+        # Question: What is this state?
         self._state = set()
```

Comment:
This is the internal variables used to know if an attribute is part of the state. We are keeping only the attribute name.